FirebaseInAppMessaging/Sources/Analytics/FIRIAMClearcutLogStorage.m

```diff
@@ -166,10 +166,22 @@ - (void)loadFromCachePath:(NSString *)cacheFilePath {
   NSString *filePath = cacheFilePath == nil ? [self.class determineCacheFilePath] : cacheFilePath;
 
   NSTimeInterval start = [self.timeFetcher currentTimestampInSeconds];
+  id fetchedClearcutRetryRecords;
+  NSData *data = [NSData dataWithContentsOfFile:filePath];
+  if (data) {
+    if (@available(macOS 10.13, iOS 11.0, tvOS 11.0, *)) {
+      fetchedClearcutRetryRecords = [NSKeyedUnarchiver
+          unarchivedObjectOfClass:[NSMutableArray<FIRIAMClearcutLogRecord *> class]
+                         fromData:data
+                            error:nil];
+    } else {
```

Comment:
Per offline discussion, please use https://developer.apple.com/documentation/foundation/nskeyedunarchiver/2962885-unarchivedobjectofclasses?language=objc to fix CI errors.